### PR TITLE
Add release branch to test build

### PR DESCRIPTION
**Pull request by CuriousVini**

### CODE REVIEW

**Comment by CuriousVini** [comment]
Please confirm the base branch. I followed this PR:
https://github.com/cdapio/cdap-build/pull/356

**Comment by CuriousVini** [comment]
Hmm looks like workflow is launched from develop branch:
image

File: .github/workflows/build-and-unit-test.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
         run: |
           if [ ${{ github.event_name }} != "workflow_dispatch" ];
           then
-            echo 'BRANCH_MATRIX={"include":[{"branch":"develop"}, {"branch":"release/6.7"}, {"branch":"release/6.8"}]}' >> $GITHUB_ENV
+            echo 'BRANCH_MATRIX={"include":[{"branch":"develop"}, {"branch":"release/6.7"}, {"branch":"release/6.8", {"branch":"release/6.9"}}]}' >> $GITHUB_ENV
```
Comment (itsankit-google) [inline .github/workflows/build-and-unit-test.yml:56]: Closing brace missing around release/6.8? 

Also we can remove release/6.7 from the list.

Comment (CuriousVini) [reply .github/workflows/build-and-unit-test.yml]: done

**Review by itsankit-google — APPROVED** [review]
Please squash commits before merge.

Thanks!

**Pull request merged**